scheduler/example-prod-config.edn

```diff
@@ -20,6 +20,7 @@
                                 :retry-limit 15
                                 :cpus 20}}
  :executor {:command "./cook-executor"
+            :portion 0.25
```

Comment:
Even if portion was permanent, having portion be set in our sample config doesn't well convey that it is used only temporarily.